docs/src/tutorials/conic/ellipse_fitting.jl

```diff
@@ -0,0 +1,416 @@
+# ellipse to a set of points using a conic optimization approach.
+
+# ## Required packages
+
+# This tutorial uses the following packages:
+
+using JuMP
+import SCS
+import LinearAlgebra
+import Images
+import Wavelets
+using LinearAlgebra
+using DSP
+using Clustering
+using Plots
+using LinearOperatorCollection
+using RegularizedLeastSquares
+
+# ## Parametrization of an ellipse
+
+# An ellipse is a set of the form:
+# ```math
+# \begin{equation}
+# \mathcal{E} = \{ \xi : (\xi - c)^\top D (\xi - c) = r^2 \}
+# \end{equation}
+# ```
+# where ``c \in \mathbb{R}^2`` is the center of the ellipse, 
+# ``D \in \mathbb{R}^{2 \times 2} \succ 0`` is a symmetric positive definite matrix and 
+# ``r > 0``.
+
+# We can setup a coordinate system ``(x, y) \in \mathbb{X} \times \mathbb{Y}`` with 
+# ``x, y \geq 0``. We use definition (1) to write an ellipse as the root of a quadratic 
+# form in homogeneous coordinates:
+# ```math
+# \begin{equation} 
+#     \begin{bmatrix} \xi \\ 1 \end{bmatrix}^T \begin{bmatrix} Q & d \\ d^T & e \end{bmatrix} \begin{bmatrix} \xi \\ 1 \end{bmatrix} = 0
+# \end{equation}
+# ```
+# Where:
+# ```math
+# \begin{align}
+#     Q &= D\\
+#     d &= -D c\\
+#     e &= c^T D c - r^2
+# \end{align}
+# ```
+# The residual distance ``r_0`` of a random point ``\xi_0 = (x_0, y_0)`` to the ellipse is 
+# then given by:
+# ```math
+# \begin{equation}
+#     r_0 \triangleq \begin{bmatrix} \xi_0 \\ 1 \end{bmatrix}^T \begin{bmatrix} Q & d \\ d^T & e \end{bmatrix} \begin{bmatrix} \xi_0 \\ 1 \end{bmatrix}
+# \end{equation}
+# ```
+# The value of ``r_0`` is positive if the point is outside the ellipse, zero if it is on
+# the ellipse and negative if it is inside the ellipse. We also see we only need 6 
+# parameters to uniquely define an ellipse.
+
+# ## Helper functions
+
+# We define some helper functions to help us visualize the results.
+# <details>
+# <summary>Functions</summary>
+function plot_dwt(x, sz = (500, 500))
+    return plt = heatmap(
+        x;
+        color = :grays,
+        aspect_ratio = 1,
+        cbar = false,
+        xlims = (0, size(x, 2)),
+        ylims = (0, size(x, 1)),
+        size = sz,
+        dpi = 300,
+    )
+end
+
+function normalize(x)
+    x = (x .- minimum(x)) ./ (maximum(x) - minimum(x))
+    return x
+end
+# </details>
+
+# ## Reading the test image
+
+# To test our ellipse-fitting algorithm we need a test image with elliptical features.
+# For our test image we will use an image of the cartwheel galaxy, 
+# [captured by the James Webb Space Telescope](https://webbtelescope.org/contents/media/images/2022/039/01G8JXN0K2VBQP112RNSQWTCTH).
+# Galaxies [come in many shapes and sizes](https://science.nasa.gov/universe/galaxies/types/), elliptical being one of them.
+
+# This is just a toy problem with little scientific value, but you can imagine how the 
+# rotation and position of elliptical galaxies can be useful information to astronomers.
+img = load("docs/src/assets/cartwheel_galaxy.png");
```

Comment:
```suggestion
img = load("../../assets/cartwheel_galaxy.png");
```
This has to be relative to the current directory. It's something like: